### ISSUE TITLE
Release notes improvements

### ISSUE THREAD

**Issue by tooomm** [freaktechnik/advanced-github-notifier#212]
Most users have auto update enabled and don't review/check all changes.

Due to the `Release Notes` only showing the most recent change(s), features or important changes are easily missed: 
![Untitled](https://user-images.githubusercontent.com/9874850/105365506-9e43db00-5bfe-11eb-90b6-b38223fdcc7f.png)

It would be helpful to have a Changelog with version numbers and some simple history there like:

> - v1.7.1 [2021-01-15]
>     - Show correct icon for draft PRs
>
> - v1.7.0 [2020-04-07]
>    - Update badge counter when opening notification
>    - Option to hide badge
>
> - v1.6.3 [2020-03-03]
>    - Fix popup populating for real now
>
> ...

**Comment by freaktechnik** [comment]
Thanks for the suggestion.

I don't like changelogs where you don't know exactly what changed in your current version because they just list a bunch of versions (unless there are unreleased/partially release versions since the last change).

However having a link to the full changelog might be an interesting idea.

**Comment by tooomm** [comment]
>you don't know exactly what changed in your current version because they just list a bunch of versions 

The list of changes in `Release Notes` should have your current version on top and don't show any changes in newer versions that are not installed/used.

<br>

>However having a link to the full changelog might be an interesting idea.

Another wording could be:

```
Changes in v1.7.1 [2021-01-15]:

  - Show correct icon for draft PRs

For a complete version history and changes in other versions check: xxxx
```

**Comment by tooomm** [comment]
I saw you added a link to the AMO version history already. 👍 

And thanks for the contributor update. :)

**Comment by tooomm** [comment]
>I saw you added a link to the AMO version history already. 👍

This is gone now again. Only the most recent change is displayed and one has no easy way to catch up on former changes.

With v1.8.0 I only get this now:
![180](https://user-images.githubusercontent.com/9874850/123088021-87134e80-d425-11eb-87d5-1d108ab6740b.png)

It used to have a link to https://addons.mozilla.org/en-GB/firefox/addon/advanced-github-notifier/versions/.

**Comment by freaktechnik** [comment]
Yes, I didn't add the link in the 1.8.0 release notes. I have to add it to every version manually if I want it to be there. And honestly I'm mostly waiting on the upstream fix for it (one day I might be tempted to do that one myself if it doesn't move at all)

**Comment by freaktechnik** [comment]
It seems I haven't linked the upstream bug here yet, it's https://bugzilla.mozilla.org/show_bug.cgi?id=1688045

**Comment by tooomm** [comment]
>I have to add it to every version manually if I want it to be there. And honestly I'm mostly waiting on the upstream fix for it (one day I might be tempted to do that one myself if it doesn't move at all)

Makes totally sense!

Thanks for taking care of the bug report (and linking it). 👍
